packages/inventory/src/services/inventory-item.ts

```diff
@@ -41,7 +47,7 @@ export default class InventoryItemService {
   async list(
     selector: FilterableInventoryItemProps = {},
     config: FindConfig<InventoryItem> = { relations: [], skip: 0, take: 10 }
-  ): Promise<InventoryItem[]> {
+  ): Promise<InventoryItemDTO[]> {
     const queryBuilder = this.getListQuery(selector, config)
```

Comment:
suggest: I think that it should be part of a custom method in the repository. Not sure it should stay in the service.
Usually, here you use the buildQuery, pass everything to the custom method repo and then you can manage the location and stuff but half of it is already managed by the utility. wdyt?

Reply:
For now I will just move it to a helper function. This module is sharing resources with core, and custom repositories should also be exported to be loaded, I have to test all that.